src/Avalonia.Themes.Default/FlyoutPresenter.xaml

```diff
@@ -0,0 +1,30 @@
+<Styles xmlns="https://github.com/avaloniaui">
+  <Style Selector="FlyoutPresenter">
```

Comment:
Default Flyout style should have some padding too, otherwise it doesn't look good out of box.
Maybe 4px?